ai_genomics/getters/crunchbase.py

```diff
@@ -0,0 +1,17 @@
+from ai_genomics import PROJECT_DIR, logger
+import pandas as pd
+
+
+def get_ai_genomics_crunchbase_org_ids() -> pd.DataFrame:
+    """Returns dataframe of AI and Genomics crunchbase organisation ids"""
+    try:
+        return pd.read_csv(
+            PROJECT_DIR / "inputs/data/crunchbase/ai_genomics_org_ids.csv",
+            index_col=0,
+            names=["cb_org_id"],
+            skiprows=[0],
+        )
+    except FileNotFoundError:
+        logger.error(
+            "FileNotFoundError: To create the missing file, run ai_genomics/analysis/crunchbase_definitions.py"
+        )
```

Comment:
Only logging an error means that the code will continue to execute after this function is called regardless of the outcome, so it will fail relatively quietly. In this case is that the user might only be alerted when they try to perform an operation on the data and trigger a `NoneType` error.

While logging has the advantage of providing useful information in the console and the ability to send logs to a logfile, raising an exception is more suitable for controlling the flow of code. There's no reason you can't have both though (more detailed explanations [here](https://stackoverflow.com/questions/51206787/between-raise-error-and-logging-the-error-which-one-is-a-better-practice-in-pyt)).

Reply:
Good point, I'll update it so that there is an error logged and the exception is raised.